waku/waku_store/rpc_codec.nim

```diff
@@ -56,11 +56,11 @@ proc decode*(
   if not ?pb.getField(1, req.requestId):
     return err(ProtobufError.missingRequiredField("request_id"))
 
-  var inclData: uint
+  var inclData: zint64
```

Comment:
```suggestion
  var inclData: hint32
```

because according to https://protobuf.dev/programming-guides/encoding/
 > Bools and enums are both encoded as if they were int32s. Bools, in particular, always encode as either 00 or 01. In Protoscope, false and true are aliases for these byte strings.